code/modules/lighting/light_source.dm

```diff
@@ -169,6 +170,11 @@
 					round(lum_b * ., LIGHTING_ROUND_VALUE)
 				)
 
+			else
+				effect_r += 0
+				effect_b += 0
+				effect_g += 0
```

Comment:
to read the code you need to know that's a list.
```variable += object``` is OBVIOUSLY a list, as you cant store objects in anything else (storing an object in an object is just the contents list)
where as ```variable += integer``` at first glance will be read as a mathematical operation.